addons/website/static/src/snippets/s_blockquote/001.scss

```diff
@@ -0,0 +1,66 @@
+.s_blockquote[data-vcss="001"] {
+    border: 0;
+    border-radius: $border-radius;
+
+    .s_blockquote_avatar {
+        width: map-get($spacers, 5);
+    }
+
+    .s_blockquote_icon, .s_blockquote_minimalist_border {
+        display: none;
+    }
+
+    // Card
+    &.s_blockquote_card {
+        .s_blockquote_infos {
+            flex-direction: row-reverse;
+            justify-content: end;
+            align-items: flex-start;
+            text-align: right;
+        }
+    }
+
+    // Minimalist
+    &.s_blockquote_minimalist {
+        border-radius: 0;
+        overflow: hidden;
+
+        .s_blockquote_minimalist_border {
+            display: block;
+            width: map-get($spacers, 1);
+        }
+
+        .s_blockquote_avatar {
+            display: none;
```

Comment:
I know that it was like in the mockup, but I think that we should always show the avatar.
If it's not needed the user can easily remove it if, while adding it where it doesn't exist could be complicated